tutor/src/screens/assignment-review/ux.js

```diff
@@ -209,6 +209,7 @@ export default class AssignmentReviewUX {
         if (extensions.length > 0) {
             await this.taskPlan.grantExtensions(extensions);
         }
+        await this.taskPlan.fetch();
```

Comment:
instead of refetching, have you investigated what `grantExtension` does with the data that's returned?  Maybe we need to update the taskPlan with it, or set a date somewhere after the request completes.

Reply:
Yeah that's why I put this into draft, refetching feels like too much when we can probably do as you say and update the data. I'll investigate more with DropAny work and update this PR with a better approach.